docs/exit-demos/206-operator-playerview-only.md

```
@@ -0,0 +1,137 @@
+  ```
+  --- FAIL: TestProjectViewStepAllowanceNeverZero (0.00s)
+      projectview_external_test.go:41: players=2 round=1 seat=0: ProjectView returned StepAllowance == 0
+  ```
+
+All three were reverted (never merged) — `main` at the SHA above carries none of them.
+
+## Fails closed
+
+The issue's own condition: *"the break-on-purpose runs assert the gate ran and inspected a non-empty package."* `check-bots-isolation.go` reports its inspected file count on every run, pass or fail — a directory holding nothing but `doc.go` exits `VACUOUS` rather than reporting a pass (`scripts/check-bots-isolation.go:145-162`). On the clean `main` SHA above:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)